benchmarks/src/test/java/zipkin2/reporter/PulsarSenderBenchmarks.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package zipkin2.reporter;
+
+import org.apache.pulsar.client.api.Consumer;
+import org.apache.pulsar.client.api.PulsarClient;
+import org.apache.pulsar.client.api.PulsarClientException;
+import org.apache.pulsar.client.api.SubscriptionInitialPosition;
+import org.apache.pulsar.client.api.SubscriptionType;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import zipkin2.reporter.internal.SenderBenchmarks;
+import zipkin2.reporter.pulsar.PulsarSender;
+
+import java.time.Duration;
+import java.util.Collections;
+
+import static org.testcontainers.utility.DockerImageName.parse;
+
+public class PulsarSenderBenchmarks extends SenderBenchmarks {
+  static final Logger LOGGER = LoggerFactory.getLogger(PulsarSenderBenchmarks.class);
+
+  static final class PulsarContainer extends GenericContainer<PulsarContainer> {
+    static final int BROKER_PORT = 6650;
+    static final int BROKER_HTTP_PORT = 8080;
+
+    PulsarContainer() {
+      super(parse("apachepulsar/pulsar:4.0.2"));
```

Comment:
let's use zipkin's image "ghcr.io/openzipkin/zipkin-pulsar:3.4.3" similar ideally same as upstream (copy/paste exact is easier for folks, if works)

https://github.com/openzipkin/zipkin/blob/e151986db4d1de1aea53618f18e4ee3acaa9ed3a/zipkin-collector/pulsar/src/test/java/zipkin2/collector/pulsar/PulsarExtension.java#L58-L75

Meanwhile, in this same PR please bump any others from `ghcr.io/openzipkin` to this version 3.4.3, so that they share base layers.